CHANGELOG.asciidoc

```diff
@@ -76,6 +76,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 *Filebeat*
 
 - Added `detect_null_bytes` selector to detect null bytes from a io.reader. {pull}9210[9210]
+- Added `syslog_host` variable to HAProxy module. {pull}9366[9366]
```

Comment:
A small description of the purpouse of the new variable could be nice, though this can be obvious on this case.

Reply:
I pushed a80f33f493a3fa87acdd8bf8d695f443d764cb6a.

Reply:
Perfect, thanks!